docs/reference/mapping/params/similarity.asciidoc

```diff
@@ -17,7 +17,7 @@ configuration are:
 
 `BM25`::
         The Okapi BM25 algorithm. The algorithm used by default in Elasticsearch and Lucene.
-        See {defguide}/pluggable-similarites.html[Pluggable Similarity Algorithms]
+        See {defguide}/pluggable-similarities.html[Pluggable Similarity Algorithms]
```

Comment:
I'm pretty sure this is correct

Reply:
The page it is linking to is https://www.elastic.co/guide/en/elasticsearch/guide/current/pluggable-similarites.html (which has a typo, but we can live with it).

Can you revert this change please?